src/htmerl_sax_utf8.erl

```diff
@@ -68,7 +68,16 @@ string(Bin, Options) ->
         end,
    M3 = maps:merge(default_state(), M2),
    Bin1 = norm_newlines(Bin, <<>>),
-   data(Bin1, M3).
+   Bin2 = norm_whitespaces(Bin1),
```

Comment:
I think putting this here will break any `pre` or other formatted tags.

Perhaps expanding `maybe_pop_text/1` to know if the whitespace can be collapsed would be a better place to do it?

Reply:
I never heard of the pre tag before. I'll try to see if I can figure out how to do that once I get a bit more familiar with the code.